src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -28,42 +24,66 @@ our @EXPORT = qw();
 our @EXPORT_OK = qw( make_figure_of_merit );
 our %EXPORT_TAGS = ( all => \@EXPORT_OK );
 
+use Inline Python => q{
+import sys
+import perl
+# this provides an interface back to the Perl namespace
+
+try:
+    from starcheck.star_probs import acq_success_prob, prob_n_acq, mag_for_p_acq
+except ImportError as err:
+    # write errors to starcheck's global warnings and STDERR
+    perl.warning("Error with Inline::Python imports \n".format(err))
+
+def _acq_success_prob(date, t_ccd, mag, color, spoiler):
+    out = acq_success_prob(date, t_ccd, mag, color, spoiler)
+    return out.tolist()
 
-our $CUM_PROB_LIMIT = -5.2;
+def _prob_n_acq(acq_probs):
+    n_acq_probs, n_or_fewer_probs = prob_n_acq(acq_probs)
+    return n_acq_probs.tolist(), n_or_fewer_probs.tolist()
+};
 
+our $CUM_PROB_LIMIT = 10 ** -5.2;
 
 sub make_figure_of_merit{
     my $c;
     my $self = shift;
     return unless ($c = $self->find_command("MP_STARCAT"));
+
     my @probs;
     my %slot_probs;
+
+    my $t_ccd = 19.0;  # ??
```

Comment:
Even when considered a requirement of review, I don't want the software to just die if run on re-open products instead of interrupt products (as in on the AUG1407A products in the regression test suite).